fastboot/instance-initializers/ember-data-fastboot.js

```diff
@@ -9,7 +9,25 @@ export function initialize(applicationInstance) {
         return store.peekAll(name).toArray();
       }).reduce((a,b) => a.concat(b), [])
         .filter(record => record.get('isLoaded'))
-        .map(record => record.serialize({ includeId: true}))
+        .map(record => {
+          const serializedRecord = record.serialize({ includeId: true});
+
+          record.eachRelationship((name, meta) => {
+            const link = record[meta.kind](name).link();
+
+            if (link) {
+              const dashName = name.dasherize();
```

Comment:
This assumes prototype extensions are turned on, better to use the explicit form like `Ember.String.dasherize(name)`.